### PR TITLE
feat(vpn): add sdk of connection_monitors

### PR DESCRIPTION
**Pull request by wangyuancheng01**
**What this PR does / why we need it**:add data_source_vpn_connection_health_checks

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: openstack/vpn/v5/connection_health_checks/requests.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package connection_health_checks
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+var requestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json", "X-Language": "en-us"},
+}
```
Comment (Jason-Zhang9309) [inline openstack/vpn/v5/connection_health_checks/requests.go:9]: this part is never used, just delete it

Comment (wangyuancheng01) [reply openstack/vpn/v5/connection_health_checks/requests.go]: down

File: openstack/vpn/v5/connection_health_checks/requests.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package connection_health_checks
```
Comment (Jason-Zhang9309) [inline openstack/vpn/v5/connection_health_checks/requests.go:1]: just use the API name in sdk, the sdk should only translate the API and do nothing more
```suggestion
package connection_monitors
```

File: openstack/vpn/v5/connection_health_checks/results.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+type commonResult struct {
+	golangsdk.Result
+}
+
+type ConnectionHealthChecks struct {
```
Comment (Jason-Zhang9309) [inline openstack/vpn/v5/connection_health_checks/results.go:11]:
```suggestion
type ConnectionMonitor struct {
```

File: openstack/vpn/v5/connection_health_checks/results.go [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	Status          string `json:"status"`
+	VpnConnectionId string `json:"vpn_connection_id"`
+	Type            string `json:"type"`
+	SourceIp        string `json:"source_ip"`
+	DestinationIp   string `json:"destination_ip"`
+	ProtoType       string `json:"proto_type"`
+}
+
+type ListResp struct {
+	ConnectionHealthChecks []ConnectionHealthChecks `json:"connection_monitors"`
```
Comment (Jason-Zhang9309) [inline openstack/vpn/v5/connection_health_checks/results.go:22]:
```suggestion
	ConnectionMonitors []ConnectionMonitor `json:"connection_monitors"`
```

File: openstack/vpn/v5/connection_health_checks/requests.go (hunk 2 above)
Comment (wangyuancheng01) [reply openstack/vpn/v5/connection_health_checks/requests.go]: down

File: openstack/vpn/v5/connection_health_checks/results.go (hunk 3 above)
Comment (wangyuancheng01) [reply openstack/vpn/v5/connection_health_checks/results.go]: down

File: openstack/vpn/v5/connection_health_checks/results.go (hunk 4 above)
Comment (wangyuancheng01) [reply openstack/vpn/v5/connection_health_checks/results.go]: down

**Pull request merged**